### PR TITLE
Create no-for-apaertheid.md

### PR DESCRIPTION
**Pull request by shushugah**

### CODE REVIEW

File: no-for-apaertheid.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: No Tech 4 Apartheid
+permalink: /no-tech-for-apartheid/
+---
+Call to Action for Tech Workers
```
Comment (ikehz) [inline no-for-apaertheid.md:6]: No Tech for Apartheid

File: no-for-apaertheid.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: No Tech 4 Apartheid
```
Comment (ikehz) [inline no-for-apaertheid.md:3]: for, not 4

File: no-for-apaertheid.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+permalink: /no-tech-for-apartheid/
+---
+Call to Action for Tech Workers
+
+“I work in tech and I am so sad/angry/scared about the current crisis in Gaza, but no one in tech seems to be talking about it.”
```
Comment (ikehz) [inline no-for-apaertheid.md:8]: Can you please replace all of this content w what's in the doc now?

Section headers should be prefixed w `##`.

Omit the bracketed paragraphs & add links?

File: no-for-apaertheid.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+1. Join thousands of tech workers around the world for the No Tech 4 Apartheid (NOTA) campaign. Sign up for  
+    1. Teach In date and access link 
+
+
+2. Organize a vigil to honor Mai and fellow tech workers impacted by the Gaza genocide. We encourage tech workers to organize an in-person vigil at their office to honor the lives of tech workers. More in this below. 
+
+
+3. If you haven’t been doing so already, contact your elected officials to demand that they endorse an immdiate and unconditional ceasefire. If you vote in the U.S., call your members of Congress to demand that they endorse an immediate and unconditional ceasefire. 
+
```
Comment (ikehz) [inline no-for-apaertheid.md:48]: Exclude after this.

**Review by ikehz — COMMENTED** [review]
Thank you so much!

File: no-for-apaertheid.md (hunk 3 above)
Comment (ikehz) [reply no-for-apaertheid.md:8]: Oh & need to add links.

**Comment by shushugah** [comment]
This is redundant

**Pull request closed without merging**